### DISCUSSION TITLE
Error received on logging into RTL: "Error: Connect Ehostunreach 10.21.21.95:2104"

### DISCUSSION THREAD

**Discussion by Lightning8Count** [Ride-The-Lightning/RTL#1176] [Troubleshooting Questions]
Hi there, hope you can help.  I am getting the following error code when I log into RTL:

`Error: Connect Ehostunreach 10.21.21.95:2104`

I can no longer see my channel balances or my on chain balances.  I am also no longer able to make on chain transactions.  

My node implementation is Umbrel 
Lightning Implementation is Core Lightning 
RTL version: Ride The Lightning (Core Lightning) Version 0.13.2-beta

Please help.

**Comment by saubyk** [comment]
Hi @Lightning8Count it appears that either your core lightning node is down or the interface daemon (c-lightning-REST) is not running.

Have you tried re-starting your device?

**Reply by saubyk** [reply]
converted this issue to a discussion, we can carry on the conversation here
